src/tool/crosshair.js

```diff
@@ -18,8 +18,8 @@ export default function() {
         },
         decorate = noop;
 
-    var x = function(d) { return d.xInDomainUnits ? xScale(d.x) : d.x; },
```

Comment:
Do these changes need to be made to the other tools? e.g. https://github.com/ScottLogic/d3fc/blob/master/src/tool/measure.js#L19